pkg/asset/manifests/gcp/cloudproviderconfig.go

```diff
@@ -61,5 +66,6 @@ node-tags       = {{$tag}}
 node-instance-prefix = {{.Global.NodeInstancePrefix}}
 external-instance-groups-prefix = {{.Global.ExternalInstanceGroupsPrefix}}
 subnetwork-name = {{.Global.SubnetworkName}}
+network-project-id = {{.Global.NetworkProjectID}}
```

Comment:
What are the implications of always passing network-project-id here? Should we put in some logic to only include it when network-project-id is not empty?